src/mgmt/osp-sah.ks

```diff
@@ -120,48 +120,60 @@ extern_bond_name="CHANGEME e.g. bond1"
 extern_boot_opts="onboot none"
 extern_bond_opts="mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1"
 extern_ifaces="CHANGEME e.g. em2 p1p2"
+extern_bond_mtu="CHANGEME e.g. 9000"
 
 # Bond to handle internal traffic
 internal_bond_name="CHANGEME e.g. bond0"
 internal_boot_opts="onboot none"
 internal_bond_opts="mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1"
 internal_ifaces="CHANGEME e.g. em1 p1p1"
+internal_bond_mtu="CHANGEME e.g. 9000"
 
 # Management
 mgmt_bond_name="CHANGEME e.g. bond0.110"
 mgmt_boot_opts="onboot none vlan"
+mgmt_bond_mtu= "1500"
 
 # Provisioning
 prov_bond_name="CHANGEME e.g. bond0.120"
 prov_boot_opts="onboot none vlan"
+prov_bond_mtu="1500"
 
 # Storage
 stor_bond_name="CHANGEME e.g. bond0.170"
 stor_boot_opts="onboot none vlan"
+stor_bond_mtu="CHANGEME e.g. 9000"
 
 # Public API
 pub_api_bond_name="CHANGEME e.g. bond1.190"
 pub_api_boot_opts="onboot none vlan"
+pub_api_bond_mtu="CHANGEME e.g. 9000"
 
 # Private API
 priv_api_bond_name="CHANGEME e.g. bond0.140"
 priv_api_boot_opts="onboot none vlan"
+priv_api_bond_mtu="CHANGEME e.g. 9000"
 
 # Define the bridges
 # Management bridge options
-br_mgmt_boot_opts="onboot static 192.168.110.12/255.255.255.0"
+br_mgmt_boot_opts="CHANGEME onboot static 192.168.110.12/255.255.255.0"
+br_mgmt_mtu="1500"
 
 # Provisioning bridge options
 br_prov_boot_opts="CHANGEME e.g. onboot static 192.168.120.12/255.255.255.0"
+br_prov_mtu="1500"
 
```

Comment:
Since 1500 is the only allowed setting for the provisioning network - should be enforced through profile

Reply:
This parameter is not exposed in the settings (ini) file.